Gopkg.toml

```diff
@@ -30,12 +30,8 @@
   name = "github.com/bluele/slack"
 
 [[constraint]]
-  name = "github.com/sensu/sensu-go"
```

Comment:
It appears that we're still importing our corev2 package from sensu-go. Is there a specific reason we're removing it from our vendor directory/dependency management?

Reply:
sensu/sensu-pagerduty-handler#14 (comment)